### ISSUE TITLE
Desktop event page – Left Column stutters on scroll on Chrome

### ISSUE THREAD

**Issue by kvngco** [ysdn-2016/ysdn-2016.github.io#108]
Lower frame rate in .gif, so imaging it more stuttery. Fine on Safari and Firefox

![screen capture on 2016-04-06 at 17-37-38](https://cloud.githubusercontent.com/assets/3757807/14333842/b0c7f10c-fc1e-11e5-93ef-52299a0e949f.gif)

**Comment by rosszurowski** [comment]
Should be fixed in eb45f2793d834f76ded8a9006c9fd03dde059a32.

This comes with a nasty technical caveat. Safari and Firefox both support [`position: sticky;`](http://caniuse.com/#feat=css-sticky) so are unaffected. The situation gives us two options for Chrome:

A. Jumpy scrolling on slower computers
B. Smooth scrolling on all computers, but text that can't be selected

Previously I chose solution A but, given that situation A is far more noticeable than B, I've switched it over to B in eb45f2793d834f76ded8a9006c9fd03dde059a32.
